packages/amplify-codegen/src/utils/getGraphQLDocPath.js

```diff
@@ -1,17 +1,11 @@
-const { join, dirname } = require('path');
+const { join } = require('path');
+const globParent = require('glob-parent');
 
-function getGraphQLDocPath(frontend, schemaLocation) {
-  const graphQLDirectory = dirname(schemaLocation);
-  let subDirectory;
-  switch (frontend) {
-    case 'android':
-      subDirectory = 'com/amazonaws/amplify/generated/graphql';
-      break;
-    default:
-      subDirectory = '';
+function getGraphQLDocPath(frontend, graphQLDirectory, includePathGlob) {
+  if (frontend === 'android') {
+    return join(graphQLDirectory, 'com/amazonaws/amplify/generated/graphql');
```

Comment:
Does this mean if the android users want to generate docs outside the graphQL directory, they cannot even if they have includePathGlob pointing to a different location?

Reply:
The android user generates code in Android Studio and uses different type generator. The docs file path is an input parameter for type generator. This part for android was previously hardcoded. Since it does not use amplify type codegen, changing it might fail in type generator. That's why I just keep it as hardcoded. We can fix it once we have an android user reporting issues. For this time, our customers are all from JS part.

Reply:
We should mention this at-least in the PR description, saying something like: `android projects will still ignore the include file path and generate the docs in graphQL directory` to document this limitation.